angular-client/src/pages/fault-page/shared/fault-type/bms-fault-type.model.ts

```diff
@@ -2,10 +2,10 @@ import { Fault } from '../indiv-fault/fault.model';
 import { FaultType, FaultTypeModel } from './fault-type.model';
 
 export class BMSFaultType implements FaultTypeModel {
-  subscribeToFaults(): Fault<FaultTypeModel> {
```

Comment:
Need to figure out if we want to implement our own subscribe stuff, or just use Subject<>